CONTRIBUTING.md

```diff
@@ -30,11 +30,15 @@ The sections below outline the steps in each case.
 
 1. (**important**) announce your plan to the rest of the community _before you start working_. This announcement should be in the form of a (new) issue;
 1. (**important**) wait until some kind of consensus is reached about your idea being a good idea;
-1. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in changes, possibly from the 'upstream' repository (follow the instructions [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [here](https://help.github.com/articles/syncing-a-fork/));
+1. if needed, fork the repository to your own Github profile and create your own feature branch off of the latest master
+   commit. While working on your feature branch, make sure to stay up to date with the master branch by pulling in
+   changes, possibly from the 'upstream' repository (follow the instructions
+   [here](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and
+   [here](https://help.github.com/articles/syncing-a-fork/));
+   - please ensure that your contribution follows our [style conventions](README.dev.md#style-conventions);
 1. make sure the existing tests still work by running `pytest`;
```

Comment:
Is the list numbering correct? I see 1., 1., 1., etc. (actually not touched from your PR, but I noticed it now)

Reply:
This gets parsed as a normal numbered list. It's just a way to avoid having to change the numbers manually when re-ordering things. (and indeed, this is how it came out of the cookie cutter).